samples/stock/demo/dark-candlestick-and-volume/demo.js

```diff
@@ -126,10 +126,24 @@
             height: '70%',
             crosshair: {
                 snap: false
+            },
+            title: {
+                text: 'Price',
+                reserveSpace: false,
+                style: {
+                    visibility: 'hidden'
+                }
```

Comment:
Is this a trick to retain screen reading? Otherwise we would normally just do `text: ''` to hide the axis title.

Reply:
Yes. Could also remove everything in `title` and add the  `yAxis.accessibility.description` and add the titles there if that is the better solution for it. When using a screen reader it has the same output.

Reply:
In that case I would vote for the 'yAxis.accessibility.description' method since it is a11y specific